pkg/apis/serving/v1/revision_lifecycle_test.go

```diff
@@ -684,3 +684,39 @@ func TestPropagateAutoscalerStatusRace(t *testing.T) {
 	apistest.CheckConditionFailed(r, RevisionConditionActive, t)
 	apistest.CheckConditionSucceeded(r, RevisionConditionReady, t)
 }
+
+func TestPropagateAutoscalerStatusReplicas(t *testing.T) {
+	r := RevisionStatus{}
+	testCases := []struct {
+		name                string
+		ps                  autoscalingv1alpha1.PodAutoscalerStatus
+		wantActualReplicas  int32
+		wantDesiredReplicas int32
+	}{{
+		name: "active PodAutoScaler",
+		ps: autoscalingv1alpha1.PodAutoscalerStatus{
+			ActualScale:  ptr.Int32(1),
+			DesiredScale: ptr.Int32(2),
+		},
+		wantActualReplicas:  1,
+		wantDesiredReplicas: 2,
+	}, {
+		name: "inactive PodAutoScaler",
+		ps: autoscalingv1alpha1.PodAutoscalerStatus{
+			DesiredScale: ptr.Int32(-1),
+		},
+		// actualReplicas: 0, desiredReplicas: 0
```

Comment:
might as well just set the `want` fields I think if we're going to have the comment here anyway